ui/src/components/Header/Header.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import './Header.css';
-import sampleProfilePic from '../../img/profile.png';
+import sampleProfilePic from '../../img/hamburger-menu.png';
+import sampleProfilePic2 from '../../img/acm-logo-final.png';
 
 /* Profile photo link  be passed in from the Page component */
 const Header = (props) => {
     return(
         <div className="header">
+            <img className="acm-logo" src={sampleProfilePic2} alt="Profile Pic"/>
             <img className="profile-pic" src={sampleProfilePic} alt="Profile Pic"/>
```

Comment:
Similarly here with the above comment about the descriptive icon names:

```suggestion
            <img className="acm-logo" src={acmLogo} alt="Profile Pic"/>
            <img className="profile-pic" src={hamburger} alt="Profile Pic"/>
```